### PR TITLE
refactor: remove unused code

### PR DESCRIPTION
**Pull request by WojtekTheWebDev**
Remove unused code.

## Description

Remove unused types and components.

## Related Issue

VBC-429

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by dhargitai — APPROVED** [review]
Hey @WojtekTheWebDev, just a minor thing:

You've put a period after almost all of your comments, but the comments in
packages/theme/pages/ResetPassword.vue
and
packages/theme/pages/Checkout/*
files are missing them.

Apart from that it looks good. 👍

File: packages/composables/src/helpers/consts.ts [hunk 1]
```diff
@@ -1,9 +1,36 @@
...
+/**
+ * Guest wishlist token.
+ */
 export const BIGCOMMERCE_GUEST_WISHLIST_TOKEN = '';
+/**
+ * Cookie max age.
+ */
 export const BIGCOMMERCE_COOKIE_MAXAGE = 60 * 60 * 24 * 7;
+/**
+ * Load wishlist fail message.
```
Comment (scernisevs) [inline packages/composables/src/helpers/consts.ts:22]: super minor and pedantic, but id suggest to replace all "fail message" with "error message"

File: packages/composables/src/useUser/index.ts [hunk 2]
```diff
@@ -8,6 +8,6 @@ import {
 import params from './params';
 
 /**
- * `useUser` composable function.
+ * Managing user sessions, credentials and registration.
```
Comment (scernisevs) [inline packages/composables/src/useUser/index.ts:11]: *suggestion:
Managing user sessions, authentication, registration and user details.

File: packages/composables/src/useUserOrderByCart/index.ts [hunk 3]
```diff
@@ -12,7 +12,7 @@ import {
 } from '..';
 
 /**
- *  Handles the shipping data for orders
+ * Managing shipping data for past and active user orders.
```
Comment (scernisevs) [inline packages/composables/src/useUserOrderByCart/index.ts:15]: *suggestion:
Managing shipping data for fulfilled and active user orders.

File: packages/composables/src/useUserOrderProducts/index.ts [hunk 4]
```diff
@@ -12,7 +12,7 @@ import {
 } from '..';
 
 /**
- *  Handles the product data for orders
+ * Managing product data for past and active user orders.
```
Comment (scernisevs) [inline packages/composables/src/useUserOrderProducts/index.ts:15]: *same suggestion as on the previous order type

**Review by scernisevs — APPROVED** [review]
LGTM with a few minor suggestions, feel free to ignore

**Pull request merged**